README.md

```diff
@@ -167,28 +167,28 @@ python run_evals_accelerate.py \
 
 Independently of the default tasks provided in `lighteval` that you will find in the `tasks_table.jsonl` file, you can use `lighteval` to evaluate models on tasks that require special processing (or have been added by the community). These tasks have their own evaluation suites and are defined as follows:
 
-* `extended`: tasks which have complex pre- or post-processing and are added by the `lighteval` maintainers. See the [`extended_tasks`](./extended_tasks) folder for examples.
+* `extended`: tasks which have complex pre- or post-processing and are added by the `lighteval` maintainers. See the [`extended_tasks`](./src/lighteval/tasks/extended_tasks) folder for examples.
```

Comment:
WDYT about showing an example with `ifeval` in the README perhaps just after we explain the different suites or putting it as a docstring somewhere? I can never quite remember the syntax and this would also give some visibility to the new tasks :)

Reply:
Good idea, I'll add that!